src/codeu/chat/server/Controller.java

```diff
@@ -40,24 +41,24 @@
   private final Model model;
   private final Uuid.Generator uuidGenerator;
   private final File persistentPath;
+  private final Admin admin;
```

Comment:
it feels like this object represents a server admin and would contain properties of an admin user. but this is actually a set of util functions to add/remove/check admins. All the functions in that class should be static functions and you shouldn't really have to create an object of it here.